bin/deploy.sh

```diff
@@ -59,7 +59,13 @@ done
 
 if [ "$PUBLISH" == 0 ]; then
     if [ ! -r "$FILE" ]; then echo "Error: Cannot read file '$FILE'. Try '$0 -h'" && exit 1; fi
-    openstack image delete "$IMG_NAME - PR#$PR_NUM"
+
+    old_image="$IMG_NAME - PR#$PR_NUM"
+    uuids=`openstack image list | grep $old_image | sed "s/| //" | awk '{print $1}'`
```

Comment:
I recommend you take a look at the format output options for that command. You can change it to a more machine readable format and also only have it output the column you need.